data/human/deep missions.txt

```diff
@@ -1502,6 +1519,44 @@ mission "Deep: Remnant 1C"
 
 
 
+# Branch D occurs when the player knows about the Remnant before the start of the mission and decided to not reveal it to the Deep researchers.
+mission "Deep: Remnant 1D"
+	landing
+	name `Analyze Series of Wormholes`
+	description `Scan the series of wormholes located in the Ember Waste`
+	source "Valhalla"
+	waypoint "Cardea"
+	waypoint "Insitor"
+	waypoint "Segesta"
+	waypoint "Aescolanus"
+	cargo "astronomical recorders" 5
+	to offer
+		has "Deep: Remnant: Keystone Research: done"
+		has "deep: knew of stones before scientists"
+		has "First Contact: Remnant: offered"
+		not "Deep: Remnant 1A: offered"
+		has "deep: did not reveal remnant"
+	on offer
+		conversation
+			`You contact Ivan after you land and he sends you directions to his research lab, located on the outskirts of the city. When you arrive you hardly have time to greet Ivan, Rayna, and the others before Ivan begins discussing what they have learned from the Quantum Keystones.`
+			`	"As you mentioned before, a single Keystone is sufficient to enter the wormhole. We've determined that there is a stabilization effect, either on the ship or the wormhole, that allows matter to cross the threshold. However, we'll need much more time to fully understand the true mechanism." Ivan asks, "Last time, you mentioned that there was an odd series of wormholes?"`
+			`	"Yes. They kind of make a train of sorts."`
+			`	"Right, definitely noteworthy. Well, we've decided that we need to know more about this these wormholes, as they could be significant. Could you make the circuit around this loop while carrying a suite of astronomical instruments? "`
+			choice
+				`	"Sure, I'm willing to do it."`
+					accept
+				`	"Sorry, you'll need to find someone else."`
+					decline
+	on visit
+		dialog `You have returned to <planet>, but you have not yet visited each system in the wormhole series. Go back and complete the set.`
+	on complete
+		event "deep: scan log research" 14
+		log `Scanned a series of wormholes beyond the wormhole in Terminus and handed the scan logs over to Ivan and his team.`
```

Comment:
```suggestion
		log `Scanned a train of wormholes in the Ember Waste and handed the scan logs over to Ivan and his team.`
```
it seemed awkward to talk about series of wormholes beyond the wormhole. Also, if we're going to make <wormhole train> an official term, reinforcing it in a few more places seems like a good idea.